Makefile

```diff
@@ -181,7 +183,12 @@ test-examples-kustomize-against-HEAD: $(MYGOBIN)/kustomize $(MYGOBIN)/mdrip
 test-examples-kustomize-against-latest-release: $(MYGOBIN)/mdrip
 	./hack/testExamplesAgainstKustomize.sh v5@$(LATEST_RELEASE)
 
-
+# Pushes dependencies in the go.work file back to go.mod files of each workspace module.
+.PHONY: workspace-sync
+workspace sync:
+	go mod tidy
```

Comment:
`go mod tidy` command is insufficient to check to update for every module due to this project being built by many modules.
`./hack/doGoMod.sh` is a useful utility to exec any command for every module.

Reply:
Yes I just realised we have the script, thanks!